src/libs/ValidationUtils.js

```diff
@@ -285,12 +286,14 @@ function validateIdentity(identity) {
  * @returns {Boolean}
  */
 function isValidUSPhone(phoneNumber = '', isCountryCodeOptional) {
-    // Remove non alphanumeric characters from the phone number
-    const sanitizedPhone = (phoneNumber || '').replace(CONST.REGEX.NON_ALPHA_NUMERIC, '');
-    const isUsPhone = isCountryCodeOptional
-        ? CONST.REGEX.US_PHONE_WITH_OPTIONAL_COUNTRY_CODE.test(sanitizedPhone) : CONST.REGEX.US_PHONE.test(sanitizedPhone);
+    let phone = phoneNumber || '';
 
-    return CONST.REGEX.PHONE_E164_PLUS.test(sanitizedPhone) && isUsPhone;
+    // If the country code is not present, we manually add it since the library needs international formatting
+    if (isCountryCodeOptional && !phone.startsWith('+')) {
+        phone = `+1${phone}`;
+    }
```

Comment:
```suggestion
    }

```

NAB: Just for readability